WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -92,10 +92,33 @@ class NotificationsViewController: UITableViewController, UIViewControllerRestor
     }()
 
     /// Notification Settings button
-    lazy var notificationSettingsButton: UIBarButtonItem = {
-        let button = UIBarButtonItem(image: .gridicon(.cog), style: .plain, target: self, action: #selector(showNotificationSettings))
-        button.accessibilityLabel = NSLocalizedString("Notification Settings", comment: "Link to Notification Settings section")
-        return button
+    lazy var settingsBarButtonItem: UIBarButtonItem = {
```

Comment:
Can you set this as `private` please? I know it wasn't before, but it should have been. As well as the new `markAllAsReadBarButtonItem` below.

Reply:
Oh no that's a good point. Updating right away